src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -1,12 +1,152 @@
-import { TextField } from '@mui/material';
+/* eslint-disable react/no-array-index-key */
+/**
+ *
+ * @summary - Event reporting page for the Wayfinder application.
+ *          - Users can submit reports about events they witness.
+ *
+ * @author  TylerMaloney
+ */
+import React, { useState, useEffect, useCallback } from 'react';
+import axios from 'axios';
+import { Button } from '../../components/Button/Button';
+import {
+  StyledReportContainer,
+  StyledReportOuterDiv,
+  StyledTextArea,
+  Section,
+  StyledSelect,
+  StyledInput,
+  StyledP,
+} from './report.styles';
+import constants from '../../constants/Constants';
+import useAppService from '../../services/app/useAppService';
 
 export default function Report() {
+  const [eventType, setEventType] = useState('');
+  const [details, setDetails] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isFormValid, setIsFormValid] = useState(false);
+  const { state } = useAppService();
+  const latitude = state.currentLocation ? state.currentLocation.lat : 49.2827;
+  const longitude = state.currentLocation ? state.currentLocation.long : -123.2;
+  const eventOptions: Array<string> = [
+    'Damaged Infrastructure',
+    'Animal Sighting',
+    'Suggestion/Complaint',
+    'Miscellaneous',
+  ];
+
+  const checkFormValidity = useCallback(() => {
+    const isEventTypeValid = !!eventType;
+    const isDetailsValid = details.length >= 10 && details.length <= 256;
+    // eslint-disable-next-line operator-linebreak
+    const isPhoneNumberValid =
+      (phoneNumber.length === 0) || (phoneNumber.length >= 10 && phoneNumber.length <= 16);
+    const isValid = isEventTypeValid && isDetailsValid && isPhoneNumberValid;
+
+    if (!isDetailsValid) {
+      setErrorMessage('Tip: Details must be between 10 and 256 characters long');
+    } else if (!isPhoneNumberValid) {
+      setErrorMessage('Tip: Phone number must be between 10 and 16 characters long, if provided');
```

Comment:
My schema doesnt have (but will need to) regex validation on the phone number, but it would be better to use a regex check to validate the phone number since it is a standardized item
For the phone number you should also provide an example of correct input
We also did Regex for the final in 128, so you can either dig into your or mine for examples of regex validation
`250 555 5555`, `(250) 555-5555`, etc